app/views/canvas/_edit.html.erb

```diff
@@ -1,9 +1,9 @@
 <div id="update-canvas"
-  data-can-create-modules="<%= can_create_modules(@experiment) ? "yes" : "no" %>"
-  data-can-edit-modules="<%= can_edit_modules(@experiment) ? "yes" : "no" %>"
+  data-can-create-modules="<%= can_manage_experiment?(@experiment) ? "yes" : "no" %>"
```

Comment:
this is horrible! Could we use here the true/false from the expression and remove this yes/no crap.

Reply:
This should be done as part of a refactoring.

Reply:
:+1: